conans/test/functional/command/test_package_test.py

```diff
@@ -148,10 +148,12 @@ def package_info(self):
             class Pkg(ConanFile):
                 def build(self):
                     info = self.deps_cpp_info["hello"]
-                    self.output.info("BUILD HELLO %s VERSION %s" % (info.name, info.version))
+                    self.output.info("BUILD HELLO %s VERSION %s" % (info.get_name('txt'),
```

Comment:
Maybe we can leave at least 1 test that will use the info.name old syntax, to make sure we are not accidentally breaking it?

Reply:
The previous test uses `info.name` (line 142). It was the one you commented on in your first review. I'll revert these two changes too. They are not needed in this PR.